image/apko/alpine-base-user-65532.yaml

```diff
@@ -0,0 +1,25 @@
+contents:
```

Comment:
nit: Imo image dir should contain OS image only, maybe create a separate top lvl dir named `container`, `apko`, etc.

Reply:
maybe we could have opinion of the code owner of the `images/` folder (@malt3) on that?

Reply:
Probably the right call. I'd add a separate folder for this since they are used for very different purposes.